src/main/java/uk/gov/justice/digital/job/S3FileTransferJob.java

```diff
@@ -41,7 +42,7 @@ public S3FileTransferJob(
 
     public static void main(String[] args) {
         logger.info("Job starting");
-        PicocliRunner.run(S3FileTransferJob.class, MicronautContext.withArgs(args));
+        PicocliRunner.run(S3FileTransferJob.class, MicronautContext.withArgs(args).registerSingleton(Clock.class, Clock.systemUTC()));
```

Comment:
This fixes the runtime exception where micronaut could not create the Clock dependency.